spyder_terminal/terminalplugin.py

```diff
@@ -182,6 +182,15 @@ def check_compatibility(self):
         if PYQT4 or PYSIDE:
             message = _('This plugin does not work with Qt 4')
             valid = False
+        if WINDOWS:
+            try:
+                import winpty
+                del winpty
+            except:
+                message = _('We found some problems while importing pywinpty. '
+                            'Please see https://github.com/spyder-ide/pywinpty'
+                            '/issues/59 for more information.')
```

Comment:
This message needs to be more informative because most people probably don't know how pywinpty is related to spyder-terminal.

I think this is better:

> Unfortunately, the library that **spyder-terminal** uses to create terminals is failing to work in your system. Therefore, this plugin will be deactivated.`<br><br>`
This usually happens on Windows 7 systems. If that's the case, please consider updating to a newer Windows version.